docs/README.md

```diff
@@ -12,14 +12,14 @@ Follow the detailed installation guide to deploy open feature operator to your l
 
 ## Configuration
 
-Configuration of the deployed sidecars is handled through the `FlagSourceConfiguration` custom resources defined at`openfeature.dev/flagsourceconfiguration` annotation of the deployed `PodSpec`.
+Configuration of the deployed sidecars is handled through the `FeatureFlagSource` custom resources defined at`openfeature.dev/featureflagsource` annotation of the deployed `PodSpec`.
```

Comment:
```suggestion
Configuration of the deployed sidecars is handled through the `FeatureFlagSource` custom resources defined at`openfeature.dev/featureflagsource` and applied as an annotation of the deployed `PodSpec`.
```
Your sentence doesn't make sense, like some word were dropped.  This is my best guess at what you meant.

Reply:
it makes sense to me -> FeatureFlagSource is referenced via `openfeature.dev/featureflagsource` annotation of the deployed Pod